src/hooks/useAddTokenToMetamask.ts

```diff
@@ -30,7 +30,10 @@ export default function useAddTokenToMetamask(token: Token | undefined): {
               address: token.addresses[chainId],
               symbol: token.symbol,
               decimals: token.decimals,
-              image: getTokenIconPath(token.symbol),
+              image:
+                token.symbol === "SDL"
+                  ? "https://assets.coingecko.com/coins/images/20476/small/edit_saddle.png?1660734429"
```

Comment:
Can we save this as a static variable up at the top somewhere?
const SDL_TOKEN_ICON_LINK = "https://assets.coingecko.com/coins/images/20476/small/edit_saddle.png?1660734429"